extensions/ql-vscode/test/pure-tests/remote-queries/markdown-generation/markdown-generation.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { generateMarkdown } from '../../../../src/remote-queries/remote-queries-markdown-generation';
+
+describe('markdown generation', async function() {
+  it('for path-problem-query: should generate markdown file for each repo with results', async function() {
```

Comment:
Minor: It might make sense to use nested ` describe` blocks. So, here you would change to:

```suggestion
  describe('for path-problem-query', () => {
    it('should generate markdown file for each repo with results', async function() {
```

And then fix indenting below. Same with the second test.